src/components/grid/grid.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../frameless";
+
+.grid {
+    display: flex;
+    padding: 12px;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+
+    .thumbnail {
+        padding: 12px;
```

Comment:
You'll want to add `display: inline-block` to this for older IE support (no flexbox).